pkg/proc/gdbserial/gdbserver.go

```diff
@@ -495,7 +509,7 @@ func LLDBAttach(pid int, path string) (*Process, error) {
 	p.conn.isDebugserver = isDebugserver
 
 	if listener != nil {
-		err = p.Listen(listener, path, pid)
+		err = p.Listen(listener, path, pid, false)
```

Comment:
Is this because the implementation is not complete, so we default to never foregrounding the process?

Reply:
It just doesn't make sense to bring the stub process to the foreground if we are attaching to a different process.

Reply:
Ah, didn't notice this was in attach not launch, my fault.